src/Employer.Web/Views/DeleteVacancy/Delete.cshtml

```diff
@@ -0,0 +1,32 @@
+﻿@model Esfa.Recruit.Employer.Web.ViewModels.DeleteVacancy.DeleteViewModel
+
+<div class="grid-row">
+    <div class="column-two-thirds">
+        <div asp-validation-summary="All" class="error-summary" role="alert" aria-labelledby="error-summary-heading-example-1" tabindex="-1"></div>
+
+        <h1 class="heading-large">@Model.Title</h1>
+        <h1 class="heading-large">Delete vacancy</h1>
+        
+        <form asp-route="@RouteNames.DeleteVacancy_Delete_Post">
+            <div class="form-group">
+                <fieldset>
+                    <legend class="heading-medium">Are you sure?</legend>
+
+                    <div class="multiple-choice">
+                        <input asp-for="@Model.ConfirmDeletion" type="radio" value="true" id="delete-yes" />
```

Comment:
Do we need to create our own `id` attribute, is it for styling? I thought the `asp-for` would generate it for us.